test/test_helper.rb

```diff
@@ -15,6 +15,20 @@
 require 'mocha/minitest'
 require 'timecop'
 require 'webrat'
+
+# Monkey patch for Nokogiri changes - https://github.com/sparklemotion/nokogiri/issues/2469
+module Webrat
+  module Matchers
+    class HaveSelector
+      def query
+          Nokogiri::CSS.parse(@expected.to_s).map do |ast|
+            ast.to_xpath("//", Nokogiri::CSS::XPathVisitor.new)
```

Comment:
An alternative to the naked `"//"` might be to reference `Nokogiri::XML::XPath::GLOBAL_SEARCH_PREFIX` in Nokogiri v1.13.0 and later.

Reply:
Because of the wide support matrix for Rails, we actually need to be able to run this with Nokogiri v1.9.1.  So I think the naked `"//"` is preferable until those Rails versions are dropped.

Reply:
:+1: That monkeypatch should work on any Nokogiri version >= v1.6.2 and < 1.13.30